stacks/earn-protocol.ts

```
@@ -0,0 +1,38 @@
+import { Api, Function, StackContext } from 'sst/constructs'
```

Biome

[error] Do not shadow the global "Function" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)